code/game/objects/structures/bonfire.dm

```diff
@@ -34,6 +34,14 @@
 /obj/structure/bonfire/permanent/sifwood/New(newloc, material_name)
 	..(newloc, MAT_SIFWOOD)
 
+/obj/structure/bonfire/examine(mob/user)
+	var/X = get_fuel_amount()
+	user << "The fire has [X] burning logs in it."
+	if(grill)
+		user << "The [src] has a crude grill plate over it."
+	if(stake)
+		user << "The [src] has a makeshift stake plate over it, perfect for witches and space templars."
```

Comment:
Undefined var for stake here. Also, please use to_chat per the new standard.

Reply:
Also, this is editing a core file. If you can, ya should make a _ch file addition or add `//CHOMPStation Addition Start/End` at either end of the new code block.

Reply:
Seconding what @TheFurryFeline said. Definitely going to need these changes before the merge could be accepted.